tests/gtest_dlt_daemon_gateway.cpp

```diff
@@ -458,6 +458,10 @@ TEST(t_dlt_gateway_parse_get_log_info, normal)
     uint8_t status = 7;
     uint16_t count_app_ids = 1;
     uint16_t count_context_ids = 1;
+#if (BYTE_ORDER == BIG_ENDIAN)
+    count_app_ids = DLT_HTOLE_16(count_app_ids);
+    count_context_ids = DLT_HTOLE_16(count_context_ids);
+#endif
```

Comment:
Hello @LocutusOfBorg ,
thanks for the patch but these IDs should not be in little endian when the MSBF bit is set.

I think we need rather a fix in the function dlt_getloginfo_conv_ascii_to_uint16_t() to reflect the endianess of the payload (as indicated by the MSBF bit)